projects/cadabra/index.js

```diff
@@ -0,0 +1,95 @@
+const {getLogs} = require("../helper/cache/getLogs");
+const {normalizeAddress} = require("../helper/tokenMapping");
+
+const BSC_VAULT = '0x6f24f5EB8f50DE36C1D1FEcD31c423e1fc8ba4ae';
+const BSC_FROM_BLOCK = 45524968;
+const BSC_ABRA_USDT_POOL = '0xa3AdcaaF941a01Eec47655c550dD5504637d029A';
+const BSC_ABRA = '0xcA1c644704feBf4ab81f85daca488d1623C28e63';
+const BSC_USDT = '0x55d398326f99059fF775485246999027B3197955';
+
+const ARBITRUM_VAULT = '0x31206FFb663651aBe29cCb72aD213d5F95BdaC45';
+const ARBITRUM_FROM_BLOCK = 293021794;
+
+async function bscTvl(api) {
+    const usdtBalance = await api.call({
```

Comment:
this the dex tvl, not yours: https://docs.llama.fi/list-your-project/what-to-include-as-tvl